xrdsst/core/validator.py

```diff
@@ -175,10 +175,10 @@ def validate_config_client_add_or_register(ss_config, operation, errors):
                 ss_config_client_slice, operation, errors
             )
 
-            require_fill(
-                ConfKeysSecServerClients.CONF_KEY_SS_CLIENT_SUBSYSTEM_CODE,
-                ss_config_client_slice, operation, errors
-            )
+            # require_fill(
```

Comment:
We should not keep code that is commented out. Maybe you should create some kind of conditional requirement here, so that when member_name is filled, subsystem_code is not required and vice versa, if subsystem_code is filled, member_name is not required, because for adding subsystems, subsystem_code is required and when adding members, member_name is required, if both are not required, it will not work